lib/diagnostic.js

```diff
@@ -38,19 +47,38 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Buddy',
+  species: 'Cat',
+  breed: 'shorthair',
+  vocalization: 'Meow',
+
+  vocalize: function() {
+    return 'Buddy says Meow';
+  },
```

Comment:
you don't need to hardcode the vocalization. 

Returning 
```js 
return this.vocalization;  
```
will return "Meow"